validator/rpc/accounts.go

```diff
@@ -173,6 +173,9 @@ func (s *Server) DeleteAccounts(
 	if err != nil {
 		return nil, err
 	}
+	if s.wallet.KeymanagerKind() != keymanager.Local && s.wallet.KeymanagerKind() != keymanager.Derived {
```

Comment:
@james-prysm are these additions safe, you think?

Reply:
this seems fine to me remote shouldn't have the ability to delete.